src/services/completions.ts

```diff
@@ -1338,13 +1338,16 @@ namespace ts.Completions {
                     case SyntaxKind.PropertyAccessExpression:
                         propertyAccessToConvert = parent as PropertyAccessExpression;
                         node = propertyAccessToConvert.expression;
-                        if ((isCallExpression(node) || isFunctionLike(node)) &&
-                            node.end === contextToken.pos &&
-                            node.getChildCount(sourceFile) &&
-                            last(node.getChildren(sourceFile)).kind !== SyntaxKind.CloseParenToken) {
+                        const leftMostAccessExpression = getLeftmostAccessExpression(parent as Expression);
+                        if (leftMostAccessExpression.pos === leftMostAccessExpression.end ||

```

Comment:
Equivalent simplifications:

```suggestion
                        const leftMostAccessExpression = getLeftmostAccessExpression(propertyAccessToConvert);
                        if (nodeIsMissing(leftMostAccessExpression) ||
```

Reply:
Commited.